python/sglang/srt/model_executor/model_runner.py

```diff
@@ -71,6 +72,8 @@
     set_cuda_arch,
 )
 
+DEFAULT_MOE_PADDING_SIZE = 8
```

Comment:
move this to cpu specific file and change it to 32.

make sure IC for each gemm on is 32x, AMX operates on tile of M16N16K32.